tests/sharing/tests/multiple_sharings.rb

```diff
@@ -2,7 +2,7 @@
 
 require_relative '../boot'
 require 'minitest/autorun'
-require 'pry-rescue/minitest'
+require 'pry-rescue/minitest' unless ENV['CI']
```

Comment:
What is it for?

Reply:
`pry-rescue/minitest` starts a pry session (the console) when an assertion fails. It's handy for debugging. But it blocks travis when the assertion fails on travis, so we require it only when we are not on a CI environment (aka travis)